### PR TITLE
IE-7761 reduce risk related to running sql user management module

### PR DESCRIPTION
**Pull request by torgeirbm**
This PR contains changes to the SqlUserManagement Powershell module. With this version, existing Sql users and their permissions are edited to match provided configuration rather than dropping all users and recreating everything. New users are created as before and dropping users is only done if input parameter is provided **and** no configuration file for the user is provided when running the module. More logging if running module with verbose option including output from sql script.
**Tested in dev environment**

### CODE REVIEW

File: scripts/sqlUserManagement/SqlUserManagement.psm1 [hunk 1]
```diff
@@ -42,6 +42,10 @@
     .PARAMETER RotationDays
     The number of days before rotating the password.
 
+    .PARAMETER DeleteUsersNotInConfiguration
+    Enable or Disable deletion of database users not included in 
+    yaml configuration. Default is do not delete.
```
Comment (MarcusNilsson) [inline scripts/sqlUserManagement/SqlUserManagement.psm1:47]: Suggestion: Add new param to readme

**Review by MarcusNilsson — APPROVED** [review]
Really nice work Torgeir! 🥇 

I took the liberty of pushing two minor changes that you can check out, and added two minor suggestions. Regardless, PR is approved!

File: scripts/sqlUserManagement/SqlUserManagement.psm1 (hunk 1 above)
Comment (torgeirbm) [reply scripts/sqlUserManagement/SqlUserManagement.psm1:47]: New input parameter added

**Review by franktore — APPROVED** [review]
Looks ok, but hard to verify simply by reading. I assume this has (or will) been tested in dev-environment

**Pull request merged**